src/Time/Units.hs

```diff
@@ -81,6 +84,7 @@ import qualified Control.Concurrent as Concurrent
 import qualified System.CPUTime as CPUTime
 import qualified System.Timeout as Timeout
 
+-- | Time stored as rational number.
```

Comment:
I think it's better to enhance documentation and say that time unit is represented as type level rational multiplier with kind `Rat`.